src/main/java/uk/gov/hmcts/reform/timedevent/infrastructure/clients/model/IdamUserInfo.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.timedevent.infrastructure.clients.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+public class IdamUserInfo {
+
+    @JsonProperty("sub")
+    private String email;
+    private String uid;
+    private List<String> roles;
+    private String name;
+
+    @JsonProperty("given_name")
```

Comment:
this is fine.  You may have already considered `@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)`

Reply:
:+1: